libsrc/include/nginterface_v2.hpp

```diff
@@ -149,6 +149,7 @@ namespace netgen
     const string * mat;   // material / boundary label
     NG_ELEMENT_TYPE GetType() const { return type; }
     int GetIndex() const { return index-1; }
+    void SetIndex(int ind) { index=ind; }
```

Comment:
I think since the GetIndex does an index shift between the netgen code and the accessing one, SetIndex should do the reverse one.

Reply:
You are correct indeed.